pkg/3scale/amp/auto-generated-templates/amp/amp-s3.yml

```diff
@@ -4548,3 +4600,9 @@ parameters:
   name: AWS_BUCKET
 - description: AWS Region to use in S3 Storage for assets.
   name: AWS_REGION
+- description: Storage Service URL scheme to use for the S3 API-compatible storage
+    service
+  name: STORAGE_SERVICE_ENDPOINT_SCHEME
+- description: Storage Service URL host and optionally a port separated by a colon
```

Comment:
Change this description to an adequate one if we change the behavior of this field